buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -136,7 +136,7 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','ppc64le_linux_xl','s390x_linux','s390x_
 echo "Initialize all PARAMETERS..."
 SETUP_LABEL = (params.SETUP_LABEL) ? params.SETUP_LABEL : "worker"
 echo "Setup SETUP_LABEL:'${SETUP_LABEL}'"
-TESTS_TARGETS = (params.TESTS_TARGETS) ? params.TESTS_TARGETS : ""
+TESTS_TARGETS = (params.TESTS_TARGETS) ? params.TESTS_TARGETS : "none"
```

Comment:
`TESTS_TARGETS` are set by `set_test_targets()` function in variables-functions.groovy. This  is not necessary.

Reply:
Done